src/components/TextArea/TextArea.jsx

```diff
@@ -35,10 +36,26 @@ const TextArea = ({
         )}
       />
       <p
-        className={classNames(styles.errorMsg, error && styles.errorMsgActive)}>
+        className={classNames(
+          styles.errorMsg,
+          error && styles.errorMsgActive,
+          wrapped && styles.noErrorMsg
+        )}>
         {error}
```

Comment:
You can get red of `noErrorMsg` and use the following:
```suggestion
        {!wrapped && error}
```